src/components/ChatEntry.css

```diff
@@ -8,6 +8,9 @@ button {
 	outline: inherit;
 }
 
+button.liked {
+}
```

Comment:
Looks like nothing got added to this rule. It happens, but it's usually worth deleting these before checking in.